Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -1998,6 +1998,38 @@ void ParseV3000AtomProps(RWMol *mol, Atom *&atom, typename T::iterator &token,
         auto ival = FileParserUtils::toInt(val);
         atom->setProp(common_properties::molStereoCare, ival);
       }
+    } else if (prop == "SUBST") {
+      if (val != "0") {
+        auto ival = FileParserUtils::toInt(val);
+        atom->setProp(common_properties::molSubstCount, ival);
+      }
+    } else if (prop == "EXACHG") {
+      if (val != "0") {
+        auto ival = FileParserUtils::toInt(val);
+        atom->setProp(common_properties::molRxnExactChange, ival);
+      }
+    } else if (prop == "INVRET") {
+      if (val != "0") {
+        auto ival = FileParserUtils::toInt(val);
+        atom->setProp(common_properties::molInversionFlag, ival);
+      }
+    } else if (prop == "ATTCHPT") {
+      if (val != "0") {
+        auto ival = FileParserUtils::toInt(val);
+        atom->setProp(common_properties::molAttachPoint, ival);
+      }
+    } else if (prop == "ATTCHORD") {
+      if (val != "0") {
+        auto ival = FileParserUtils::toInt(val);
+        atom->setProp(common_properties::molAttachOrder, ival);
+      }
+    } else if (prop == "CLASS") {
+      atom->setProp(common_properties::molAtomClass, val);
+    } else if (prop == "SEQID") {
+      if (val != "0") {
+        auto ival = FileParserUtils::toInt(val);
+        atom->setProp(common_properties::molAtomSeqId, ival);
+      }
```

Comment:
Should we check the values of these inputs?

Reply:
We're inconsistent about this. In parts of the parser we check values in others not.
I'm happy to add it if others think it's important.

Reply:
I just noticed that you checked INVRET on output, and wanted to raise the question if we should check the other cases too.

I think that, as long as we don't do anything with them, it is ok no to check, and just forward whatever we read (right or wrong). SUBST would be the exception to not using the other properties, as we use it to generate the query, but the check is already implicit in the generation of the query.